src/openrct2-ui/windows/TileInspector.cpp

```diff
@@ -1944,10 +2021,33 @@ static void window_tile_inspector_paint(rct_window* w, rct_drawpixelinfo* dpi)
                 screenCoords.y = w->windowPos.y + w->widgets[WIDX_TRACK_SPINNER_HEIGHT].top;
                 gfx_draw_string_left(dpi, STR_TILE_INSPECTOR_BASE_HEIGHT_FULL, nullptr, COLOUR_WHITE, screenCoords);
 
+                // brakes/booster speed label
+                if (tileElement->AsTrack())
+                {
+                    if (tileElement->AsTrack()->GetTrackType() == TRACK_ELEM_BRAKES
+                        || tileElement->AsTrack()->GetTrackType() == TRACK_ELEM_BOOSTER)
+                    {
+                        screenCoords.y = w->windowPos.y + w->widgets[WIDX_TRACK_SPINNER_SPEED].top;
+                        gfx_draw_string_left(dpi, STR_TILE_INSPECTOR_TRACK_SPEED, nullptr, COLOUR_WHITE, screenCoords);
+
+                        // brakes/booster speed spinner value
+                        screenCoords.x = w->windowPos.x + w->widgets[WIDX_TRACK_SPINNER_SPEED].left + 3;
+                        screenCoords.y = w->windowPos.y + w->widgets[WIDX_TRACK_SPINNER_SPEED].top;
+                        int32_t speed = static_cast<int32_t>(tileElement->AsTrack()->GetBrakeBoosterSpeed());
+
+                        // convert the speed to a readable format
+                        auto ft = Formatter().Add<int32_t>(speed);
```

Comment:
You're going to hit the copy constructor bug if the optimiser wants to mess with this. Make this two separate calls. I'm going to fix the copy constructor as well at some point but for now split this up.